src/triage/component/postmodeling/__init__.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf-8
+
+import os
+import yaml
+import logging
+import warnings
+
+import sqlalchemy
+from sqlalchemy.engine.url import URL
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+import sqlalchemy.dialects.postgresql as postgresql
+from sqlalchemy import Column, ForeignKey
+
+import pandas as pd
+
+from triage.component.catwalk.storage import CSVMatrixStore, HDFMatrixStore, Store, ProjectStorage
+
+logging.basicConfig(level=logging.INFO)
+
+
+def __db_url_from_environment():
+    environ_url = os.getenv('DATABASE_URL')
+    if environ_url:
+        logging.info("Getting db connection credentials from DATABASE_URL")
+        return environ_url
+
+    if os.getenv('DATABASE_FILE') and os.path.isfile(os.getenv('DATABASE_FILE')):
+        logging.info("Getting db connection credentials from DATABASE_FILE")
+        dbfile = open(os.getenv('DATABASE_FILE'))
+    else:
+        warnings.warn("There is no DATABASE_URL or DATABASE_FILE  environment variable")
+        return None
+
+    with dbfile:
+        dbconfig = yaml.load(dbfile)
+
+    return URL(
+        'postgres',
+        host=dbconfig['host'],
+        username=dbconfig['user'],
+        database=dbconfig['db'],
+        password=dbconfig['pass'],
+        port=dbconfig['port'],
+    )
+
+
+def create_session(engine=None):
+    if engine:
+        __engine = engine
+    else:
+        url = __db_url_from_environment()
+        if url:
+            __engine = create_engine()
+        else:
+            return None
+
+    session = sessionmaker(bind=__engine)()
+
+    return session
+
+session = create_session()
+
+Base = declarative_base()
+
+def get_model(model_id):
+    return session.query(Model).get(model_id)
+
+
+def get_model_group(model_group_id):
+    return session.query(ModelGroup).get(model_group_id)
+
+
+def get_predictions(model):
+    predictions = pd.DataFrame([prediction.__dict__ for prediction in model.predictions])
+    return predictions.drop('_sa_instance_state', axis=1).set_index(['as_of_date', 'entity_id'])
+
+
+def get_evaluations(model):
+    evaluations =  pd.DataFrame([evaluation.__dict__ for evaluation in model.evaluations])
+    evaluations['model_group_id'] = model.model_group_id
+    return evaluations.drop('_sa_instance_state', axis=1).set_index(['model_group_id', 'model_id', 'metric', 'parameter'])
+
+
+class ModelGroup(Base):
```

Comment:
Could the original definitions in https://github.com/dssg/triage/blob/master/src/triage/component/results_schema/schema.py be used? It would be nice to minimize the code that needs to be changed when the schema changes

Reply:
I will use them :)